python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -59,67 +52,87 @@ def core_info(engine):
     """
     import sgtk
     from sgtk.commands.core_upgrade import TankCoreUpdater
-    logger = sgtk.LogManager.get_logger(LOGGER_NAME)
-
-    code_css_block = "display: block; padding: 0.5em 1em; border: 1px solid #bebab0; background: #faf8f0;"
 
     # Create an upgrader instance that we can query if the install is up to date.
     installer = TankCoreUpdater(
         engine.sgtk.pipeline_configuration.get_install_location(),
-        logger,
+        engine._log,
```

Comment:
Yeah, using sgtk.log here is fine because we know it exists since where the ones who put it there in the first place during `pre_engine_start_callback`.